src/api/app/views/webui/users/token_triggers/show.html.haml

```diff
@@ -13,7 +13,7 @@
             = @token.id
           .form-group
             = f.label(:type, 'Operation:')
-            = @token.class.token_name.capitalize
+            = @token.capitalize
```

Comment:
Is it correct? Shouldn't it be `@token.name.capitalize` ?

Reply:
yes, it is correct. We are displaying the "Operation" there, so it will be "Service", "Workflow", "Release or "Rebuild" and not the custom name given by the user.

Reply:
I understand it's not the name given by the user but the "type of token". But, shouldn't the code be like:

```suggestion
            = @token.to_s.capitalize
```

Reply:
if you do something like: `puts Token.first` ruby knows it should try to call `to_s` on that object. We normally ask for changes asking to  avoid the explicit "to_s", no?

Reply:
I checked it and failed, but not in the context of a view.


```
[11] pry(main)> token.to_s.capitalize
=> "Workflow"
[12] pry(main)> token.capitalize
NoMethodError: undefined method `capitalize' for #<Token::Workflow id: 1, string: "awfaoh4m99jlm3q0vehya6eth1twj23l", user_id: 1, package_id: nil, type: "Token::Workflow", scm_token: "nfx7ldgle29ra23td1v57axxwf56n30f", name: "Testing token", triggered_at: nil>
from /usr/lib64/ruby/gems/3.1.0/gems/activemodel-6.1.4.6/lib/active_model/attribute_methods.rb:469:in `method_missing'
```

I'll check it there too.

Reply:
It also crashes when I go to the token edit view:

![Uploading Screenshot 2022-04-08 at 15-27-47 Action Controller Exception caught.png…]()

Reply:
> if you do something like: `puts Token.first` ruby knows it should try to call `to_s` on that object. We normally ask for changes asking to avoid the explicit "to_s", no?

I think what you mean here is what we do when we interpolate. For example, `"#{token}"` is equivalent to `"#{token.to_s}"
`, so we avoid specifying `.to_s` as it is redundant.